addons/project_todo/views/project_task_views.xml

```diff
@@ -50,17 +48,16 @@
                                 <div class="o_kanban_record_body o_todo_kanban_card_body" t-att-class="['1_done', '1_canceled'].includes(record.state.raw_value) ? 'opacity-50' : ''">
                                     <field name="tag_ids" widget="many2many_tags" options="{'color_field': 'color'}"/>
                                 </div>
-                                <div class="o_kanban_record_bottom">
-                                    <div class="oe_kanban_bottom_left"/>
-                                    <div class="oe_kanban_bottom_right"  t-att-class="['1_done', '1_canceled'].includes(record.state.raw_value) ? 'opacity-50' : ''">
-                                        <div t-attf-class="d-flex #{todoHasAssignees ? 'w-100 align-items-center justify-content-end' : 'align-items-end'}">
-                                            <div class="d-flex align-items-center me-2 mt-2">
-                                                <div t-if="todoHasAssignees">
-                                                    <field name="user_ids" widget="many2many_avatar_user"/>
-                                                </div>
-                                                <field name="activity_ids" widget="kanban_activity"/>
-                                            </div>
+                                <div class="o_kanban_record_bottom mt-2">
+                                    <div class="oe_kanban_bottom_left" t-att-class="['1_done', '1_canceled'].includes(record.state.raw_value) ? 'opacity-50' : ''">
+                                        <field name="priority" widget="priority"/>
+                                        <field name="activity_ids" widget="kanban_activity" style="padding-top: 1.5px;"/>
+                                    </div>
+                                    <div class="oe_kanban_bottom_right">
+                                        <div t-if="todoHasAssignees" t-att-class="['1_done', '1_canceled'].includes(record.state.raw_value) ? 'opacity-50' : ''">
```

Comment:
Problem here is that the quick assign button is not visible if there is only one assignee. Here is my suggestion to only hide the avatar and not the quick assigne button:
```suggestion
                                        <div t-att-class="{'opacity-50': ['1_done', '1_canceled'].includes(record.state.raw_value), 'o_hide_avatar': !todoHasAssignees}">
```
And in the style sheet something like this:
`.o_hide_avatar .o_avatar {
    display: none;
}`

Reply:
Hello @huca-odoo  ,
I have done the required changes.
Thank You:)